public/app/core/utils/explore.ts

```diff
@@ -225,22 +226,18 @@ export function generateKey(index = 0): string {
   return `Q-${Date.now()}-${Math.random()}-${index}`;
 }
 
-export function generateRefId(index = 0): string {
-  return `${index + 1}`;
-}
-
-export function generateEmptyQuery(index = 0): { refId: string; key: string } {
-  return { refId: generateRefId(index), key: generateKey(index) };
+export function generateEmptyQuery(queries: DataQuery[], index = 0): { refId: string; key: string } {
```

Comment:
should this not return a DataQuery?

Reply:
Hm, did not consider the return type of this function but yes it could return a DataQuery. Pushed fix.